### ISSUE TITLE
Post files get put into the wrong folders

### ISSUE THREAD

**Issue by etec-masterofsynapse** [mapleweekend/easy-reddit-downloader#14]
I noticed that files do not get put into their right directories.

Here is the log:
```
Welcome to Reddit Post Downloader!
Contribute @ https://github.com/mapleweekend/easy-reddit-downloader
What subreddit would you like to download?You may submit multiple separated by commas (no spaces).
         cloudflare,freenas,truenas,sysadmin
How many posts do you want to go through?(more posts = more downloads, but takes longer)
         1000
How would you like to sort? (top, new, hot, rising, controversial)
         new
What time period? (hour, day, week, month, year, all)
         all
How often should this be run?
1.) one time
2.) every 30 seconds
3.) every minute
4.) every 5 minutes
5.) every 30 minutes
6.) every hour
7.) every 3 hours
8.) every day
        ` 1
Requesting posts from
      https://www.reddit.com/r/cloudflare/new.json?sort=new&t=all&limit=1000
Requesting posts from
      https://www.reddit.com/r/freenas/new.json?sort=new&t=all&limit=1000
Requesting posts from
      https://www.reddit.com/r/truenas/new.json?sort=new&t=all&limit=1000
Requesting posts from
      https://www.reddit.com/r/sysadmin/new.json?sort=new&t=all&limit=1000
```

For example, one post from sysadmin went into the truenas folder, and one post from cloudflare did too. There may be more but these are two I noticed right away.

**Comment by mapleweekend** [comment]
That is bizarre! Taking a look 👀

**Comment by mapleweekend** [comment]
Fixed the bug with the latest PR #16. I also found that video links just weren't downloading at all (i.e. youtube.com links). Thanks for the bug report and helping me keep this working in tip-top shape :) 

Please report back if it's still broken after downloading the latest repo
